BookingsApi/.vscode/launch.json

```diff
@@ -0,0 +1,7 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
please add this to git ignore to avoid checking this in

Reply:
done

Reply:
please remove it from the PR too so it doesn't get merged in

Reply:
done